### PR TITLE
Network partition: add partition_one

### PR DESCRIPTION
**Pull request by mahjonp**
Signed-off-by: mahjonp <junpeng.man@gmail.com>

### What problem does this PR solve? 

partition_one isolates one node from other nodes

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: cmd/util/suit.go [hunk 1]
```diff
@@ -9,11 +9,12 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/pingcap/tipocket/pkg/nemesis"
```
Comment (mapleFU) [inline cmd/util/suit.go]: Why do you move it here...

File: pkg/nemesis/network_partition.go [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+					Pods: map[string][]string{
+						anotherPart[0].Namespace: extractPodNames(anotherPart),
+					},
+				},
+				TargetMode: chaosv1alpha1.AllPodMode,
+			},
+		},
+	})
+}
+
```
Comment (mapleFU) [inline pkg/nemesis/network_partition.go]: The `NetworkChaos` templates is almost the same...why not just create a function like `podTag`

Comment (mahjonp) [reply pkg/nemesis/network_partition.go]: IMO, it's early to extract such function

File: cmd/util/suit.go (hunk 1 above)
Comment (mahjonp) [reply cmd/util/suit.go]: fixed

File: pkg/nemesis/network_partition.go [hunk 3]
```diff
@@ -0,0 +1,165 @@
+package nemesis
+
+import (
+	"context"
+	"fmt"
+	uuid "github.com/satori/go.uuid"
```
Comment (mapleFU) [inline pkg/nemesis/network_partition.go]: group imports

**Comment by you06** [comment]
I have the question about kill nemesis, it kills randomly from all nodes, including tidb node. Does the bank case work when killing tidb?

**Comment by mahjonp** [comment]
> I have the question about kill nemesis, it kills randomly from all nodes, including tidb node. Does the bank case work when killing tidb?

Yeah, I think BankClient can solve such connection error.

File: pkg/nemesis/network_partition.go (hunk 3 above)
Comment (mahjonp) [reply pkg/nemesis/network_partition.go]: resolved

**Pull request merged**